packages/components/pie-assistive-text/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@justeattakeaway/pie-assistive-text",
+  "description": "PIE Design System Assistive Text built using Web Components",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "custom-elements.json",
+    "src",
+    "dist",
+    "**/*.d.ts"
+  ],
+  "scripts": {
+    "build": "npx build-react-wrapper && run -T vite build",
+    "create:manifest": "yarn cem analyze --litelement",
+    "lint:scripts": "run -T eslint .",
+    "lint:scripts:fix": "yarn lint:scripts --fix",
+    "lint:style": "run -T stylelint ./src/**/*.{css,scss}",
+    "lint:style:fix": "yarn lint:style --fix",
+    "watch": "run -T vite build --watch",
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "test:ci": "yarn test",
+    "test:browsers": "npx playwright test -c ./playwright-lit.config.ts",
+    "test:browsers:ci": "yarn test:browsers",
+    "test:visual": "echo \"Error: A new Percy project needs to be created before visual tests can be run. steps: https://vue.pie.design/?path=/story/documentation-standards-testing-visual-regression--page \" && exit 1",
```

Comment:
i think you need to update the test:visual command here to follow these [steps](https://www.figma.com/file/OOgnT2oNMdGFytj5AanYvt/%5BCore%5D-Web-Component-Documentation-%5BPIE-3%5D?node-id=1148%3A14397&mode=dev)

Reply:
you will also need to update the project label configs as mentioned[ in the readme](https://github.com/justeattakeaway/pie/tree/main/packages/tools/generator-pie-component#setting-up-the-project-label-for-github)

Reply:
oh sorry didn't realise I had to do that in this PR, it's already added to percy so can do that now 👍

Reply:
and lastly, u will need to add the component to storybook as mentioned [here](https://github.com/justeattakeaway/pie/tree/main/packages/tools/generator-pie-component#setting-up-storybook)

Reply:
here is how a full example should look [like](https://github.com/justeattakeaway/pie/pull/1229), happy to hop on a call if it's easier 🤝